airbyte-cdk/python/README.md

```diff
@@ -14,8 +14,7 @@ This document is a general introduction to the CDK. Readers should have basic fa
 Generate an empty connector using the code generator. First clone the Airbyte repository then from the repository root run
 ```
 cd airbyte-integrations/connector-templates/generator
```

Comment:
Can we make it run from the project root instead?

```
./airbyte-integrations/connector-templates/generator/generate.sh
```

We've had this convention for almost all our scripts.

Reply:
I move the generate.sh to the root of the project, update the script paths, and READMEs. So now it can be started from the airbyte root.